server/src/main/java/org/opensearch/cluster/node/DiscoveryNode.java

```diff
@@ -390,7 +390,7 @@ public void writeTo(StreamOutput out) throws IOException {
                 .collect(Collectors.toList());
             out.writeVInt(rolesToWrite.size());
             for (final DiscoveryNodeRole role : rolesToWrite) {
-                if (role == DiscoveryNodeRole.MASTER_ROLE) {
+                if (role == DiscoveryNodeRole.MASTER_ROLE || role == DiscoveryNodeRole.CLUSTER_MANAGER_ROLE) {
```

Comment:
Hi @reta, thanks a lot for your review! 👍👍 I will think about it and make the change. Since the original code was using `==`, I didn't think further 😂

Reply:
I updated in the commit https://github.com/opensearch-project/OpenSearch/pull/2424/commits/7a2a6b99374d0b2cc6eefc72ef4087e0004a395a, changed all the `==` to `.equals()` in this file (when comparing with with `DiscoveryNodeRole`).

Reply:
:+1: